transport/src/main/java/io/netty/channel/SliceableFileRegion.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2017 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.channel;
+
+import java.io.IOException;
+import java.nio.channels.WritableByteChannel;
+
+/**
+ * A {@link FileRegion} which supports slice.
+ * <p>
+ * We introduce this interface is only because that we can not add new methods to an interface until
```

Comment:
"is only because that" seems to have a grammar issue. Maybe: "We introduce this interface because we..."

Reply:
Done.